### ISSUE TITLE
actix-web-actors incompatible with actix-web 3.3.2

### ISSUE THREAD

**Issue by rthomas** [actix/actix-web#2212]
## Expected Behavior
Examples build successfully.

## Current Behavior
Using `actix-web-actors` `3.0.0` (latest crates.io release) with `actix-web` `3.3.2` we get a compilation error in the websocket (`actix/examples/websockets/websocket`) example.

The examples build correctly with the versions specified in the examples - but fail with the latest release of `actix-web` and `actix-web-actors`.

To reproduce:
1. Checkout the examples repository
2. `cd websockets/websocket`
3. Run `cargo upgrade`
4. Run `cargo build`

```
error[E0277]: the trait bound `MyWebSocket: actix::actor::Actor` is not satisfied
   --> websockets/websocket/src/main.rs:79:23
    |
79  |     fn hb(&self, ctx: &mut <Self as Actor>::Context) {
    |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `actix::actor::Actor` is not implemented for `MyWebSocket`
    | 
   ::: /home/ryan/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-web-actors-3.0.0/src/ws.rs:181:8
    |
181 |     A: Actor<Context = WebsocketContext<A>>,
    |        ------------------------------------ required by this bound in `WebsocketContext`

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0277`.
error: build failed
```

## Possible Solution

I'm unsure if `actix-web-actors` just needs to be released again - it has been 8 months since the last crates.io release.

## Context

I have been attempting to implement websockets in an `actix-web` webapp, using `cargo-edit` to add the latest version of the `actix-web` and `actix-web-actors` dependencies lead me to this issue in my project, which I reproduced in the examples repo.

## Your Environment

linux 5.12.0-1-MANJARO #1 SMP PREEMPT Mon Apr 26 22:09:28 UTC 2021 x86_64 GNU/Linux
rustc 1.52.1 (9bc8c42bb 2021-05-09)
cargo 1.52.0 (69767412a 2021-04-21)
actix-web 3.3.2
actix-web-actors 3.0.0

**Comment by rthomas** [comment]
I tracked this down, it appears to be an incompatability with `actix` `0.11` - moving back to `actix` `0.10` as a dependency fixes this.

Please close if this is working as intended.

**Comment by rthomas** [comment]
@robjtede should a new release of `actix-web-actors` be published with support for `actix` `0.11`? It's been ~8 months since the last release.
